oat-sa/extension-tao-itemqti
# Fix/tr 1420 fix mobile safari tap highlight

Related to https://oat-sa.atlassian.net/browse/TR-1420

- Remove mobile safari tap highlight color

Test:
- Install this version of taoQtiItem into the mentioned client environment
- Import the test mentioned in the ticket
- Try to check answer in mobile safari (xcode emulator works as well)
